rotkehlchen/premium/premium.py

```diff
@@ -207,6 +211,75 @@ def set_credentials(self, credentials: PremiumCredentials) -> None:
             self.reset_credentials(old_credentials)
             raise PremiumAuthenticationError('rotki API key was rejected by server')
 
+    def get_remote_devices_information(self) -> dict:
+        """Get the list of devices for the current user"""
+        method = 'manage/premium/devices'
+        data = self.sign(
+            method=method,
+            api_endpoint='webapi',
+        )
+
+        try:
+            response = self.session.get(
+                f'{self.rotki_web}{method}',
+                data=data,
+                timeout=ROTKEHLCHEN_SERVER_TIMEOUT,
+            )
+        except requests.exceptions.RequestException as e:
+            msg = f'Could not connect to rotki server due to {e!s}'
+            log.error(msg)
+            raise RemoteError(msg) from e
+
+        data = _process_dict_response(response)
+        return data
+
+    def authenticate_device(self) -> None:
+        """
+        Check if the device is in the list of the devices and if it isn't add it when possible.
+        May raise:
+        - PremiumAuthenticationError: when the device can't be registered
```

Comment:
Shouldn't you also add a remote error here? There is two different calls to rotki server in this function